test/code/plugins/patch_management_lib_test.rb

```diff
@@ -204,25 +204,25 @@ def test_available_updates_transform_and_wrap
             "DataItems" => [{
                 "Host" => "HostName",
                 "AgentId" => "AgentId-123",
-                "OSFullName" => "Ubuntu 16.04",
+                "OSFullName" => "Ubuntu 15.04",
                 "OSName" => "Ubuntu",
                 "OSType" => "Linux",
-                "OSVersion" => "16.04",
+                "OSVersion" => "15.04",
                 "Timestamp" => "2016-03-15T19:02:38.577Z",
                 "Collections" => [{
-                    "CollectionName" => "dpkg_1.18.4ubuntu1.1_Ubuntu_16.04",
+                    "CollectionName" => "dpkg_1.18.4ubuntu1.1_Ubuntu_14.04",
                     "Installed" => false,
                     "PackageName" => "dpkg",
                     "PackageVersion" => "1.18.4ubuntu1.1",
-                    "Repository" => "Ubuntu:16.04/xenial-updates",
+                    "Repository" => "Ubuntu:15.04/xenial-updates",
                     "Timestamp" => "1970-01-01T00:00:00.000Z"
                 }]
             }]
         }
     expectedTime = Time.utc(2016,3,15,19,2,38.5776)
     wrappedHash = LinuxUpdates::transform_and_wrap(@available_updates_xml_str, "HostName", expectedTime,
-                                                   86400, "AgentId-123", "Ubuntu", "Ubuntu 16.04",
-                                                   "16.04", "Ubuntu_16.04")
+                                                   86400, "AgentId-123", "Ubuntu", "Ubuntu 15.04",
```

Comment:
The dependency injection pattern would be worth to take a look at for future testing needs.